Packs/CyberArk/CHANGELOG.md

```diff
@@ -0,0 +1 @@
+## [Unreleased]
```

Comment:
this file is not needed anymore. if `demisto-sdk init` creates it - please open an issue.